modular/gater/router.go

```diff
@@ -262,6 +263,9 @@ func (g *GateModular) RegisterHandler(router *mux.Router) {
 	// List Bucket Read Quota Count
 	router.Path("/").Name(getBucketReadQuotaCountRouterName).Methods(http.MethodGet).Queries(ListBucketReadCountQuery, "").HandlerFunc(g.getBucketReadQuotaCountHandler)
 
+	// Get BsDB data statistics Info
+	router.Path("/").Name(getBsDBDataInfo).Methods(http.MethodGet).Queries(BsDBInfo, "").HandlerFunc(g.getBsDBDataInfoHandler)
```

Comment:
To improve naming consistency, consider changing `BsDBInfo` to `BsDBInfoQuery`, similar to `ListBucketReadCountQuery` and `GetPaymentByBucketIDQuery`, to clearly indicate its use for routing.